flake.nix

```diff
@@ -131,6 +135,11 @@
             ...
           }:
           {
+            # For standardised reproducible formatting with `nix fmt`
+            # HELP: why does the per system formatter not work?
+            # formatter = inputs.nixfmt.packages.${system}.nixfmt;
```

Comment:
This doesn't work, and I couldn't quite figure out why. From the error message it appears to try building the aarch_64 darwin build in x86_64-linux.

```
       error: a 'aarch64-darwin' with features {} is required to build '/nix/store/apv47ywrf96wyiy5cd9l9grz0kmg1y1y-cabal2nix-nixfmt.drv', but I am a 'x86_64-linux' with features {benchmark,
 big-parallel, kvm, nixos-test, uid-range}
```

Reply:
Oh, I see. Do you see a solution that does not require changes in `nixfmt`'s flake? Else maybe we can use nixfmt from nixpkgs, plus point being we get the builds cached for free.

Reply:
(sorry, can't suggest anything rn, but can review your proposals; I'm more available next week)